libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -203,8 +203,8 @@ string YulUtilFunctions::leftAlignFunction(Type const& _type)
 			break;
 		case Type::Category::Enum:
 		{
-			unsigned storageBytes = dynamic_cast<EnumType const&>(_type).storageBytes();
-			templ("body", "aligned := " + leftAlignFunction(IntegerType(8 * storageBytes)) + "(value)");
+			solAssert(dynamic_cast<EnumType const&>(_type).numberOfMembers() <= 256, "");
+			templ("body", "aligned := " + leftAlignFunction(IntegerType(8)) + "(value)");
 			break;
```

Comment:
I changed the code in `Types` and `YulUtilFunctions` so that it assumes that enums are 1-byte long. As you can see this assumption does not simplify the code all that much. Should I keep these changes? The PR would work just fine without them. Longer enums would then be possible internally but not allowed in the syntax.

Reply:
I'm fine with either.